tests/utils.py

```diff
@@ -49,3 +51,99 @@
     '_id': 'yes! yes! yes!',
     'count': 0
 }
+
+
+TEST_SCHEMA = update_schema(BASEXMLSCHEMA, {
+    "properties": {
+        "title1": "//dc:title/node()",
+        "title2": ["//dc:title/node()", lambda x: x.lower()],
+        "title3": ["//dc:title/node()", "//dc:title/node()", lambda x, y: x + y.lower()]
```

Comment:
I think the downside would be that specifying named selectors would be too verbose to do inline, which would complicate the schema specification. I'm not sure that named selectors would necessarily make the code more readable.

This is what I imagine the usage looking like, and in both situations I think just having args is still simpler to read.

```python

def process_title(t, title=None):
    return t.lower() + title.lower() if title else t.lower()

title2_kwargs = {
    "title": "//dc:title/node()"
}

SCHEMA = update_schema(BASEXMLSCHEMA, {
    "properties": {
        "title1": (("//dc:title/node(),), {"title1": "//dc:title/node()"}), process_title),
        "title2": (("//dc:title/node(),), title2_kwargs), process_title)
    }
)
```